tests/features/search/europa_search.feature

```diff
@@ -4,6 +4,7 @@ Feature: Europa Search features
   As a anonymous user
   I want to search content from a form available on the site
 
+  @wip
```

Comment:
To relate it to NEPT-1172: I changed it in the PR 1473 with the right tag to use.